File: java_free_map_backend_system/app/src/main/kotlin/grpc_key_string_maps/account_storage_service.kt

package grpc_key_string_maps

class account_storage_service_key_string_maps {
    companion object {
            class CreateUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class CreateUserResponse {
                companion object {
                    @JvmField 
                    var result: String = "result"
                    var error: String = "error"
                    val __property_list__: List<String> = listOf("result", "error")
                }
            }

            class DeleteUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class DeleteUserResponse {
                companion object {
                    @JvmField 
                    var result: String = "result"
                    var error: String = "error"
                    val __property_list__: List<String> = listOf("result", "error")
                }
            }

            class UpdateUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    var username: String = "username"
                    var head_image: String = "head_image"
                    var sex: String = "sex"
                    var age: String = "age"
                    val __property_list__: List<String> = listOf("email", "username", "head_image", "sex", "age")
                }
            }

            class UpdateUserResponse {
                companion object {
                    @JvmField 
                    var result: String = "result"
                    var error: String = "error"
                    val __property_list__: List<String> = listOf("result", "error")
                }
            }

            class GetUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class GetUserResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var user_exists: String = "user_exists"
                    var email: String = "email"
                    var username: String = "username"
                    var head_image: String = "head_image"
                    var sex: String = "sex"
                    var age: String = "age"
                    val __property_list__: List<String> = listOf("error", "user_exists", "email", "username", "head_image", "sex", "age")
                }
            }

            class UserModel {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    var username: String = "username"
                    var head_image: String = "head_image"
                    var sex: String = "sex"
                    var age: String = "age"
                    val __property_list__: List<String> = listOf("email", "username", "head_image", "sex", "age")
                }
            }
    }
}

File: python_chat_with_friends_system/src/grpc_key_string_maps/account_storage_service.py

from typing import List

class CreateUserRequest:
    email: str = "email"
    __property_list__: List[str] = ["email"]

class CreateUserResponse:
    result: str = "result"
    error: str = "error"
    __property_list__: List[str] = ["result", "error"]

class DeleteUserRequest:
    email: str = "email"
    __property_list__: List[str] = ["email"]

class DeleteUserResponse:
    result: str = "result"
    error: str = "error"
    __property_list__: List[str] = ["result", "error"]

class UpdateUserRequest:
    email: str = "email"
    username: str = "username"
    head_image: str = "head_image"
    sex: str = "sex"
    age: str = "age"
    __property_list__: List[str] = ["email", "username", "head_image", "sex", "age"]

class UpdateUserResponse:
    result: str = "result"
    error: str = "error"
    __property_list__: List[str] = ["result", "error"]

class GetUserRequest:
    email: str = "email"
    __property_list__: List[str] = ["email"]

class GetUserResponse:
    error: str = "error"
    user_exists: str = "user_exists"
    email: str = "email"
    username: str = "username"
    head_image: str = "head_image"
    sex: str = "sex"
    age: str = "age"
    __property_list__: List[str] = ["error", "user_exists", "email", "username", "head_image", "sex", "age"]

class UserModel:
    email: str = "email"
    username: str = "username"
    head_image: str = "head_image"
    sex: str = "sex"
    age: str = "age"
    __property_list__: List[str] = ["email", "username", "head_image", "sex", "age"]